src/App.tsx

```diff
@@ -50,22 +51,102 @@ import ContentManager from "./api/ContentManager"
 import { Plugins, AppState } from '@capacitor/core';
 import NotificationsAssistant from "./api/NotificationsAssistant"
 
+
+
 // init content manager
 let cm = new ContentManager();
 
 const App: React.FC = () => {
 
-  // capacitor notifications setup
-  useIonViewDidEnter(() => {
-    // refresh notifications on app enter
-    Plugins.App.addListener("appStateChange", (state: AppState) => {
+
+  useEffect( () => {
+
+    console.log("Starting app")
+
+    let notificationAssistant = new NotificationsAssistant();
+
+    let remain:Boolean;
+    let starting:Number;
+    let ending:Number;
+
+    notificationAssistant.NoNotificationsRemaining().then((remaining) => {
```

Comment:
The goal is to always reload all the notifications. You shouldn't be checking for whether or not we have a few left, I think

Reply:
Well it mainly checks to see if we have no notifications, and if there isn't anything, we just do nothing

Reply:
If there are any remaining notifications it just calls the NotificationSetter, which will clear any pending notifications

Reply:
So then why are we doing nothing if we run out of notifications? Isn't that a great time to also add notifications?